github/libp2p/repository_collaborator.json

```diff
@@ -468,1276 +135,373 @@
+      "permission": "admin"
     },
     "web3-bot": {
       "permission": "push"
     }
   },
-  "js-peer-info": {
-    "daviddias": {
+  "go-smart-record": {
+    "petar": {
       "permission": "admin"
     },
-    "dignifiedquire": {
-      "permission": "push"
-    },
-    "nunofmn": {
+    "web3-bot": {
       "permission": "push"
     }
   },
-  "jvm-libp2p": {
+  "go-sockaddr": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "libp2p": {
-    "Mr0grog": {
-      "permission": "pull"
-    },
-    "carospiegly": {
-      "permission": "pull"
-    },
-    "raulk": {
-      "permission": "admin"
-    },
+  "go-socket-activation": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "mplex": {
-    "aarshkshah1992": {
+  "go-tcp-transport": {
+    "web3-bot": {
       "permission": "push"
     }
   },
-  "notes": {
-    "aarshkshah1992": {
-      "permission": "push"
-    },
-    "jsoares": {
-      "permission": "push"
-    },
+  "go-ws-transport": {
     "web3-bot": {
       "permission": "push"
-    },
-    "yiannisbot": {
+    }
+  },
+  "go-yamux": {
+    "web3-bot": {
       "permission": "push"
     }
   },
-  "npm-go-libp2p": {
-    "achingbrain": {
+  "gossipsub-hardening": {
+    "vyzo": {
       "permission": "admin"
     }
   },
-  "npm-go-libp2p-dep": {
-    "aarshkshah1992": {
-      "permission": "push"
-    },
-    "vasco-santos": {
+  "hydra-booster-infra": {
+    "guseggert": {
       "permission": "admin"
-    },
+    }
+  },
+  "js-libp2p-bootstrap": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "observation-deck": {
-    "AlanSl": {
-      "permission": "admin"
-    },
-    "mcollina": {
-      "permission": "admin"
-    },
-    "raulk": {
-      "permission": "admin"
-    },
+  "js-libp2p-crypto": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "observer-toolkit": {
-    "AlanSl": {
-      "permission": "admin"
-    },
-    "aarshkshah1992": {
+  "js-libp2p-daemon": {
+    "web3-bot": {
       "permission": "push"
-    },
-    "mcollina": {
+    }
+  },
+  "js-libp2p-daemon-client": {
+    "vasco-santos": {
       "permission": "admin"
-    },
-    "raulk": {
+    }
+  },
+  "js-libp2p-daemon-protocol": {
+    "achingbrain": {
       "permission": "admin"
-    },
+    }
+  },
+  "js-libp2p-delegated-content-routing": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "projects": {
-    "aarshkshah1992": {
-      "permission": "push"
-    },
+  "js-libp2p-delegated-peer-routing": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "pubsub-notes": {
+  "js-libp2p-floodsub": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "pull-mplex": {
+  "js-libp2p-interfaces": {
     "web3-bot": {
       "permission": "push"
     }
   },
-  "py-libp2p": {
-    "alexh": {
-      "permission": "push"
-    },
-    "carver": {
-      "permission": "admin"
-    },
-    "ralexstokes": {
-      "permission": "admin"
-    },
```

Comment:
I agree that I'm not active. I don't have any special desire to be a maintainer, though there have been times in the past when there was no one available/capable to take care of maintenance and I was happy to do some work. Sometimes that work has required admin access (like adding a readthedocs url hook to fix the doc builds).

My main concern here is that it seems like *everyone* is being removed from the repo. Presumably I'm missing something about how permissions are working here. I would just worry about getting things working quickly in some sort of emergency.

If you feel like leaving me on as a backup maintainer, I'm cool with that. Otherwise, I'd just ask that it's really clear who we can communicate with, in the case of an urgent need.